pkg/server/container_stop.go

```diff
@@ -82,13 +84,15 @@ func (c *criContainerdService) StopContainer(ctx context.Context, r *runtime.Sto
 	}
 	glog.Errorf("Stop container %q timed out: %v", id, err)
 
-	glog.V(2).Infof("Delete container from containerd %q", id)
-	_, err = c.containerService.Delete(ctx, &execution.DeleteRequest{ID: id})
+	glog.V(2).Infof("Kill container %q", id)
+	_, err = c.containerService.Kill(ctx, &execution.KillRequest{ID: id, Signal: uint32(unix.SIGKILL)})
```

Comment:
I think I've missed something. Where do you `Delete` the container after this change?